mesop/dataclass_utils/diff_state_test.py

```diff
@@ -395,6 +396,32 @@ class C:
   ]
 
 
+def test_diff_uploaded_file():
```

Comment:
add a test case when the uploaded file is the same? See earlier comment, it's not clear whether the two uploaded file with the same value (but different identity) will produce a diff or not.

Reply:
Sure added a test to check same file data case (different identity)